composer.json

```diff
@@ -62,8 +62,8 @@
     "symfony/css-selector": "4.4.*",
     "symfony/maker-bundle": "~1",
     "symfony/panther": "^0.9",
-    "symfony/profiler-pack": "~1",
-    "symfony/test-pack": "^1.0"
+    "symfony/phpunit-bridge": "~5",
+    "symfony/profiler-pack": "~1"
```

Comment:
`test-pack` was swapped out for `phpunit-bridge`. Was this intentional. If so, I'd like to hear why. Other than that, thanks for installing the updates!

:100: